test/integration/local_transactions_test.rb

```diff
@@ -117,6 +117,17 @@ class LocalTransactionsTest < ActionDispatch::IntegrationTest
         )
       end
 
+      should "add the GA4 link tracker around the result link (information_click)" do
+        data_module = page.find("#get-started")["data-module"]
+        expected_data_module = "ga4-link-tracker"
+
+        ga4_link_attribute = page.find("#get-started")["data-ga4-link"]
+        ga4_expected_object = "{\"event_name\":\"information_click\",\"type\":\"local transaction\",\"tool_name\":\"Pay your bear tax\",\"action\":\"information click\"}"
+
+        assert_equal expected_data_module, data_module
+        assert_equal ga4_expected_object, ga4_link_attribute
+      end
```

Comment:
Could some of these commits that add a test be merged with their appropriate code change commit?